crd-generator/api/src/test/java/io/fabric8/crd/example/nocyclic/NoCyclicStatus.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.crd.example.nocyclic;
+
+public class NoCyclicStatus {
+  private String message;
+  // private Ref ref1;
```

Comment:
This doesn't pass yet, and I'm not sure how to implement it without breaking the signature of public methods, any hint? Or any suggestion?
@iocanel maybe?

Reply:
fixed